IntegrationAdapters/MyDbContext.cs

```diff
@@ -0,0 +1,21 @@
+﻿using IntegrationAdapters.Models;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace IntegrationAdapters
+{
+    public class MyDbContext : DbContext

```

Comment:
PharmacyApiContext is a much more suitable name 😄

Reply:
But MyDbContext will contain other DbSets (tables) in the future, not only Apis

Reply:
Do you mean for the whole Hospital database? In that case it can be HealthCareSystemContext